geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -129,6 +130,8 @@ app.MeasureController = function($scope, $q, $http, ngeoDecorateInteraction,
   this.map_ = this['map'];
 
   var measureProfile = new ngeo.interaction.MeasureLength({
+    startMsg: goog.dom.getElement('measure-profile-start-message'),
```

Comment:
And it looks like we do extract translation strings from js files: https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/CONST_Makefile#L438.